test/e2e/metering_manual_install_test.go

```diff
@@ -86,8 +90,8 @@ func testManualMeteringInstall(
 	}
 
 	rf, err := deployerCtx.Setup(deployerCtx.Deployer.InstallOLM, expectInstallErr)
-	canSafelyRunTest := testhelpers.AssertCanSafelyRunReportingTests(t, err, expectInstallErr, expectInstallErrMsg)
```

Comment:
Revert this change too.

Reply:
Strange, I don't have that line subtracted. It is in the file. So going to leave alone

Reply:
I think Tim means the whitespace change v. master is not needed, the location of the empty line was changed.